lib/runtime/src/component/endpoint.rs

```diff
@@ -17,6 +17,8 @@ use derive_getters::Dissolve;
 
 use super::*;
 
+pub use async_nats::service::endpoint::Stats as EndpointStats;
```

Comment:
Renaming types makes it hard for the person reading the code. Does it conflict?

Reply:
There are actually two different types of stats in nats... service::Stats and endpoint::Stats.

I was using this to differentiate the two.

We only expose endpoint::Stats to the user.